homeassistant/components/geniushub/__init__.py

```diff
@@ -0,0 +1,54 @@
+"""This module connects to the Genius hub and shares the data."""
+import logging
+
+import voluptuous as vol
+
+from homeassistant.const import (
+    CONF_HOST, CONF_PASSWORD, CONF_USERNAME)
+from homeassistant.helpers import config_validation as cv
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+from homeassistant.helpers.discovery import async_load_platform
+
+REQUIREMENTS = ['geniushub-client==0.3.4']
```

Comment:
Remove this. Moving forward we only specify requirements on the manifest.json file.

Reply:
Done.

Reply:
Done